.github/workflows/tests.yaml

```diff
@@ -98,6 +98,21 @@ jobs:
                      --door-lock-app ./out/linux-x64-door-lock-${BUILD_VARIANT}/chip-door-lock-app \
                      --tv-app ./out/linux-x64-tv-app-${BUILD_VARIANT}/chip-tv-app \
                   "
+            - name: Build and run OTA example
+              timeout-minutes: 30
+              run: |
+                  rm -rf /tmp/ota/
```

Comment:
Does it make sense for these `rm -rf` to go inside the `ota_test.sh`?

Reply:
removed

Reply:
I guess it does, since Linux and Darwin make the same call. This would help remove some duplicated code.

Reply:
Thanks, saw that you added to ota_test.sh but the rm/mkdir are still here. As is the case for Darwin below.

Reply:
Ah, forgot to push that file thank you

Reply:
Looking at it again, I'm not certain Uploading OTA logs will be put in the right place, since now the /tmp/ota directory is created after the log path is specified. I will and see what happens.